common/models/workspace.js

```
@@ -169,6 +178,17 @@ module.exports = function(Workspace) {
         return cb(err);
       }
 
+      var defaultDependencies = template.package.dependencies;
+      if (loopbackVersion === '2.x') {
+        template.package.dependencies =
+          extend(defaultDependencies, DEPENDENCY_2_X);
+      } else if (loopbackVersion === '3.x') {
+        template.package.dependencies =
+          extend(defaultDependencies, DEPENDENCY_3_X);
+      } else {
+        throw new Error('Loopback version should be either 2.x or 3.x');
```

Should we return the error via callback instead?